### PR TITLE
DateTime fix return condition

### PR DESCRIPTION
**Pull request by titkov49**
Other changes:
- Remove `style-loader`. `MiniCssExtractPlugin.loader` is adding style element to html instead.
- Add title with distance from date to DateTime
- Remove `package.json` from `asab-webui` root
- Add defense from invalid dates

### CODE REVIEW

**Comment by titkov49** [comment]
@Pe5h4 
https://user-images.githubusercontent.com/37152497/155122002-64533df3-05d7-4471-9e09-8f4ac3f36991.mov

File: src/containers/DateTime.js [hunk 1]
```diff
@@ -30,23 +30,54 @@ The default format is `lll` -> `Aug 22, 2020 1:13 PM`
...
+		);
 	}
 
-	const m = isNaN(props.value) || props.value > 9999999999 ? moment(props.value) : moment(props.value * 1000);
-	m.locale(language.slice(0, 2));
+	if (new Date(props.value).toString() === "Invalid Date") {
+		return (
+			<span className='datetime'>
+				<i className="cil-clock pr-1"></i>
+				Invalid Date
```
Comment (Pe5h4) [inline src/containers/DateTime.js:64]: @titkov49 Is it possible to translate it?

**Review by Pe5h4 — APPROVED** [review]
Looks cool. Just one question mentioned above - if it would be possible, please update

File: src/containers/DateTime.js (hunk 1 above)
Comment (titkov49) [reply src/containers/DateTime.js:64]: > @titkov49 Is it possible to translate it?

@Pe5h4 I can edit it as common translation:
```js
t("Invalid Date")
```

**Pull request merged**